questions/unit-testing.md

```diff
@@ -0,0 +1,22 @@
+### What is Unit Testing ?
+
+### Answer
+
+Unit Testing is a level of software testing where individual units or components of software are tested independently.
+
+Tests are written using frameworks like Mocha, Chai, and Jest. The software is validated by giving it different inputs, and checking to see that the desired output is returned.
+
+### Good to hear
+
+**Benefits**:
```

Comment:
you can remove the benefits bold here entirely now, since the benefits are the good to hear